modules/serverless-build-func-kn.adoc

```diff
@@ -39,7 +62,21 @@ Building function image
 Function image has been built, image: quay.io/username/example-function:latest
 ----
 
-To learn more about `kn func build` command options, you can use the help command:
+[id="serverless-build-func-kn-push_{context}"]
+== Push flag
+
+You can add the `--push` flag to a `kn func build` command to attempt to automatically push the function image after it is successfully built:
+
+.Example build command using OpenShift Container Registry
+[source,terminal]
+----
+$ kn func build --push
+----
+
+[id="serverless-build-func-kn-help_{context}"]
+== Help command
+
+You can use the help command to learn more about `kn func build` command options:
```

Comment:
This is probably a stupid question, but is the help command something users would know? Is it a flag like `--push` Not sure, but it might be worth adding the command to this section. Depends on if you think users would know it already or not.

Reply:
oh whoops I see it is there just not part of this PR. Disregard the previous comment. lol